streams/src/main/java/org/apache/kafka/streams/state/internals/WindowStoreBuilder.java

```diff
@@ -36,7 +36,6 @@ public WindowStoreBuilder(final WindowBytesStoreSupplier storeSupplier,
                               final Serde<V> valueSerde,
                               final Time time) {
         super(storeSupplier.name(), keySerde, valueSerde, time);
-        Objects.requireNonNull(storeSupplier, "storeSupplier can't be null");
```

Comment:
Could you elaborate why this is not required?

Reply:
@abbccdda It will throw a exception at line 38 if `storeSupplier ` is null.  So line 39 is redundant.

Reply:
@tang7526 Thanks, could we do sth similar to https://github.com/apache/kafka/blob/d30b4e51513e6938970020bf1e894c983447ef8f/streams/src/main/java/org/apache/kafka/streams/state/internals/SessionStoreBuilder.java#L36 here?